src/BinSkim.Rules/PERules/BA2025.EnableShadowStack.cs

```diff
@@ -25,6 +25,10 @@ public class EnableShadowStack : WindowsBinaryAndPdbSkimmerBase
         /// Control-flow Enforcement Technology (CET) Shadow Stack is a computer processor feature
         /// that provides capabilities to defend against return-oriented programming (ROP) based
         /// malware attacks.
+        /// Note: older versions of .NET are not compatible with CET/shadow stack technology. 
```

Comment:
@shaopeng-gh This is really a note for the users right? Why add it as a comment here? It does not affect the analysis in any way correct?

Reply:
correct it does not affect the analysis in any way, our current pattern is to add these strings as comment in the code, so developer don't need to open the resource string and find what it is, easier to understand.
And we keep these comments in sync with the actual resource string.

Reply:
Understood. All good then.